build-tools/tasks/test-utils.js

```diff
@@ -18,12 +19,59 @@ const configs = {
   dom: {
     defaultExport: `export default function wrapper(root: Element = document.body) { if (document && document.body && !document.body.contains(root)) { console.warn('[AwsUi] [test-utils] provided element is not part of the document body, interactions may work incorrectly')}; return new ElementWrapper(root); }`,
     buildFinderInterface: ({ componentName }) =>
-      `find${componentName}(selector?: string): ${toWrapper(componentName)} | null;`,
+      `/**
+        * Returns the wrapper of the first ${componentName} that matches the specified CSS selector.
+        * If no CSS selector specified, returns the wrapper of the first ${componentName}.
+        * If no matching ${componentName} is found, returns \`null\`.
+        *
+        * @param {string} [selector] CSS Selector
+        * @returns {${toWrapper(componentName)} | null}
+        */
+       find${componentName}(selector?: string): ${toWrapper(componentName)} | null;
+       /**
+        * Returns the wrappers of all ${pluralize(componentName)} that match the specified CSS selector.
+        * If no CSS selector specified, returns all of the ${pluralize(componentName)} inside current wrapper.
+        * If no matching ${componentName} is found, returns an empty array.
+        *
+        * @returns {Array<${toWrapper(componentName)}>}
+        */
+       findAll${pluralize(componentName)}(selector?: string): Array<${toWrapper(componentName)}>;
+       /**
+        * Returns the first ${componentName} that matches the specified test id.
```

Comment:
I'd recommend to make "id" uppercase, it is in my experience the more common way of writing it
```suggestion
        * Returns the first ${componentName} that matches the specified test ID.
```